.env

```diff
@@ -1 +1 @@
-NEXT_PUBLIC_NETWORKS=Ethereum=http://localhost:8080
+NEXT_PUBLIC_NETWORKS=Bitcoin=http://bitcoin:8080,Ethereum=http://ethereum:8081,Polkadot=http://substrate:8082
```

Comment:
not very pleased with having to make this change. but the issue is that in release mode the environment variables need to be set at compile time. would much prefer to be able to set them at runtime in my docker-compose file